language-server/modules/langserver-core/src/test/resources/hover/configs/hover_remote_method_decl2.json

```diff
@@ -1,21 +1 @@
-{
```

Comment:
Are these changes OK? We used to format those previously.

Reply:
This is rectified